### DISCUSSION TITLE
P instead of Check text when saying Pass.

### DISCUSSION THREAD

**Discussion by jkchua** [getgauge/gauge#2262] [General]
This may be a simple one, but may I ask how do I change P into a ✔️ in the command line? I noticed from the website around Step Implementation that the page says ✔️  instead of the reality, which is P. 

![image](https://user-images.githubusercontent.com/11019850/178409603-d3d68b34-7fd3-452e-b7aa-5f195e5a784d.png)

Is that a Gauge configuration one? If so, what should I change around. If it matters, I am using Java and Maven 😊

**Comment by sriv** [comment]
you can pass `--simple-console` flag to gauge to get a simpler console output (without the colour and ticks/crosses). 

In maven, you can do this by setting the `flags` property either at invocation or in your pom.xml

**Reply by jkchua** [reply]
I think I am asking for the opposite 😊 I want those check marks instead of letter P. When I used the `--simple-console` command, the P (or the check box as well) disappeared. 😄 It applies as well on Powershell, Terminal via VS Code. I am thinking this may be due to the fact that I am using Maven instead of NodeJS? 

![image](https://user-images.githubusercontent.com/11019850/178626273-e6664b60-d74e-48e7-8283-70aeaef1a92b.png)

Thanks for the idea of configuration of flags though 😄
